index.html

```diff
@@ -1,31 +1,32 @@
 <!doctype html>
 <html>
-  <head>
-    <link rel='stylesheet' href='styles.css'>
-  </head>
-  <body>
-    <header>
-      <h1>The Header</h1>
-    </header>
-    <aside>
-      <h2>A Side Bar</h2>
-    </aside>
-    <div class='main'>
-      <h2>The Main Content</h2>
-      <div class='left'>
-	This is the left side
-	<img src='https://jesse.sh/img/me.jpg'>
-      </div>
-      <div class='right'>
-	This is the right side
-      </div>
-    </div>
-    <aside>
-      <h2>A Side Bar</h2>
-    </aside>
-    <footer>
-      <h3>A Footer</h3>
-      <p>&copy; GA</p>
-    </footer>
-  </body>
+<head>
+  <title>Checkpoint 1</title>
+  <link rel='stylesheet' type="text/css" href='styles.css'>
+</head>
+<body>
+  <header>
+    <h1>The Header</h1>
+  </header>
+  <aside>
+    <h2>A Side Bar</h2>
+  </aside>
+  <main>
+    <h2>The Main Content</h2>
+  This is the left side
+  <div class="left">
+    <img src='https://jesse.sh/img/me.jpg'/>
```

Comment:
👍  was thinking the alt tag but will count this (keep in mind `alt` tags are more important than closing slashes which some people like but have no effect in HTML5)